themes/beaver/static/css/newblog.css

```diff
@@ -0,0 +1,478 @@
+    @media (prefers-color-scheme: dark) {
+        body {
+            background: #000
```

Comment:
use consistent styles: run in IDE to reformat this code to make consistent style

Reply:
++